jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -298,7 +300,8 @@ object XGBoost extends Serializable {
       obj: ObjectiveTrait = null,
       eval: EvalTrait = null,
       useExternalMemory: Boolean = false,
-      missing: Float = Float.NaN): XGBoostModel = {
+      missing: Float = Float.NaN,
+      checkIntervals: Long = 30 * 60 * 1000): XGBoostModel = {
```

Comment:
we should put it in `params` to reduce the API signature length

Reply:
Moved.